presto-product-tests/conf/presto/etc/log.properties

```diff
@@ -6,7 +6,7 @@
 #
 
 io.prestosql=INFO
+io.prestosql.server.PluginManager=WARN
```

Comment:
i agree this is too verbose.
I am concern WARN may be too little.

maybe we got `PluginManager` log levels incorrectly in https://github.com/prestosql/presto/pull/1311?

Reply:
I agree, it seems too verbose to me.